choose/choose.go

```diff
@@ -133,28 +127,12 @@ type item struct {
 	order    int
 }
 
-func (m model) Init() tea.Cmd {
-	return timeout.Init(m.timeout, nil)
-}
+func (m model) Init() tea.Cmd { return nil }
 
 func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.WindowSizeMsg:
 		return m, nil
-	case timeout.TickTimeoutMsg:
```

Comment:
Is this a breaking change?

Reply:
shouldn't be no

Reply:
FWIW `--timeout` isn't even working in latest gum version  :P